docs/modules/getting_started/examples/code/getting-started.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# The getting started guide script
+# It uses tagged regions which are included in the documentation
+# https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/
+#
+# There are two variants to go through the guide - using stackablectl or helm
+# The script takes either 'stackablectl' or 'helm' as an argument
+#
+# The script can be run as a test as well, to make sure that the tutorial works
+# It includes some assertions throughout, and at the end especially.
+
+if [ $# -eq 0 ]
+then
+  echo "Installation method argument ('helm' or 'stackablectl') required."
+  exit 1
+fi
+
+case "$1" in
+"helm")
+echo "Adding 'stackable-dev' Helm Chart repository"
+# tag::helm-add-repo[]
+helm repo add stackable-dev https://repo.stackable.tech/repository/helm-dev/
+# end::helm-add-repo[]
+
+echo "Installing Operators with Helm"
+# tag::helm-install-operators[]
+helm install --wait commons-operator stackable-dev/commons-operator --devel
```

Comment:
The --devel is a good way to get the dev version in Helm without specifying the version explicitly.

But once we release this Operator version with this version of the docs, we need to put in a stable version here. So I think it makes sense to already have the versions here and put them in the templating file, so later it's easy to change the version (once we release)

Reply:
done